docker-micro-benchmark/benchmark.sh

```diff
@@ -37,14 +37,23 @@ doBenchmark() {
   $DOCKER_MICRO_BENCHMARK $1 > $RDIR/result_benchmark.dat &
   BENCHMARK_PID=$!
   $STAT_TOOL -p $BENCHMARK_PID 1 > $RDIR/cpu_benchmark.dat &
-  DOCKER_PID=`ps -ef | awk '$8=="/usr/bin/docker" {print $2}'`
+  DOCKER_PID=`pidof dockerd`
+  if [ $? -ne 0 ]; then
+    # Be compatible with old version.
+    DOCKER_PID=`pidof docker`
+  fi
   $STAT_TOOL -p $DOCKER_PID 1 > $RDIR/cpu_docker_daemon.dat &
   DOCKER_PIDSTAT=$!
+  # For old version without containerd, there is no data for containerd,
+  # and the corresponding data won't be plotted.
+  CONTAINERD_PID=`pidof docker-containerd`
+  $STAT_TOOL -p $CONTAINERD_PID 1 > $RDIR/cpu_containerd_daemon.dat &
```

Comment:
* If we add check here, `cpu_containerd_daemon.dat` will not be generated, and we'll get a warning`warning: Skipping unreadable file "cpu_containerd_daemon.dat"`
* If we don't check here, an empty `cpu_containerd_daemon.dat` will be generated, and we'll get a warning `warning: Skipping data file with no valid points`.

None of them will affect the result, and they will only happen for docker <= 1.10, which should be acceptable. :)